src/router/routes/mobile.js

```diff
@@ -25,6 +25,8 @@ import SendAmount from '../../pages/mobile/SendAmount.vue';
 import SendConfirm from '../../pages/mobile/SendConfirm.vue';
 import TransactionList from '../../pages/mobile/TransactionList.vue';
 import TransactionDetails from '../../pages/mobile/TransactionDetails.vue';
+import Names from '../../pages/mobile/Names.vue';
+import NamesNew from '../../pages/mobile/NamesNew.vue';
```

Comment:
I propose to use the singular form of Names (NameList, NameNew) as used in Network, RemoteConnection and so on